lib/commands.js

```diff
@@ -547,10 +548,13 @@ exports.bindCommands = function bindCommands(forum) {
                 if (!checkAvailable(forbiddenCmds, cmd, 'error', forbiddenMsg)) {
                     return reject(new Error('E_FORBIDDEN_COMMAND'));
                 }
+                if (context === undefined) {
+                    context = new Object();
```

Comment:
Is that a breaking change? I'm not sure what the old value of `this` would have been.